e2e/helpers/utils.ts

```diff
@@ -1,11 +1,25 @@
 import { Page, expect } from '@playwright/test'
 
+export async function waitForSrcEndpointConfig(page: Page) {
+  // window only available after page has initially loaded
+  await navigateToHomepageIfPageHasNotBeenLoaded(page)
+  // ensure that endpoint config is set,
+  // ...so API calls point to the correct stack
+  await expect(async () => {
+    const response = await page.evaluate('window.SRC.OVERRIDE_ENDPOINT_CONFIG')
+    expect(response).not.toBeUndefined()
+  }).toPass()
+}
+
 // Use after initially navigating to baseURL
 // ...to give SWC time to compile, if necessary
 export async function waitForInitialPageLoad(page: Page) {
   await expect(page.getByRole('heading', { name: 'Loading…' })).not.toBeVisible(
     { timeout: 2 * 60 * 1000 }, // ...wait 2 minutes if necessary
   )
+  // ensure that the SRC endpoint has been set
+  // ...before navigation initiates API calls
+  await waitForSrcEndpointConfig(page)
```

Comment:
Ensure that SRC endpoint has been set before navigating away from homepage -- hope this resolves occasional flakiness when navigating to dashboard, as in [this run](https://github.com/Sage-Bionetworks/SynapseWebClient/actions/runs/6127124526/job/16632396261), where Projects button doesn't appear after clicking on 'View Your Dashboard'.